src/controllers/Question.controller.js

```diff
@@ -11,6 +11,35 @@ const createQuestion = (question) => {
   });
 };
 
+const addQuestionChoices = (question) => new Promise((resolve, reject) => {
+    if(question){
+        ChoiceController.getChoiceByQuestionId(question.id)
+            .then((choices) => {
+                resolve({ question, choices });
+            })
+            .catch((err) => {
+                reject(err);
+            })
+    } else {
+        reject(new Error('No question found'));
+    }
+
+});
+
+
+const getQuestionById = (req, res) => {
+    const {questionId} = req.params;
+    Question.findOne({where: {id: questionId}})
+        .then(addQuestionChoices)
```

Comment:
`Question.findOne({where: {id: questionId},
include: [{
      model: Choice,
},attributes: [ 'id',    'text',
    'imageUrl',
    'type',
    'quizNo',
    'category',, 'choice.text' ] )`
If you do joint as the above you wouldn't need to use the `addQuestionChoices`